docs/src/atoms/promptExample.jsx

```diff
@@ -0,0 +1,26 @@
+import {
+    A,
+    Button,
+    DropdownButton,
+    Prompt,
+} from 'react-cm-ui';
+import React from 'react';
+
+function PrompExample() {
```

Comment:
Misspelled `PromptExample` here (missing the "t" in "Prompt").

Reply:
Thanks